pom.xml

```diff
@@ -21,10 +21,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-security</artifactId>
-        </dependency>
+<!--        <dependency>-->
+<!--            <groupId>org.springframework.boot</groupId>-->
```

Comment:
If you want to comment it out - there should be a //TODO comment added with a Jira note ID of a task implementing the security. If you leave it as is it could stay forever in here as a redundant comment.